website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -110,17 +110,34 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
  
-### SQLFluff integration
-The dbt Cloud CLI supports [SQLFluff](https://sqlfluff.com/), a modular and configuration SQL linter, which warns you of complex functions, syntax, formatting, and compilation errors.
+### Lint SQL files 
 
-To get started, run `dbt sqlfluff -h` to see the list of supported commands and flags, such as `dbt sqlfluff lint` to lint SQL files.
+From the dbt Cloud CLI, you can invoke [SQLFluff](https://sqlfluff.com/) which is a modular and configurable SQL linter that warns you of complex functions, syntax, formatting, and compilation errors. Many of the same flags that you can pass to SQLFluff are available from the dbt Cloud CLI, except those that might pose a security risk.
+
+The available SQLFluff commands are: 
+
+- `lint` &mdash; Lint SQL files by passing a list of files or from standard input (stdin).
+- `fix` &mdash; Fix SQL files.
+- `format` &mdash; Autoformat SQL files.
+
+
+To lint SQL files, run the command as follows:  
+
+```shell
+dbt sqlfluff lint [PATHS]... [flags]
+```
+
+When no path is set, dbt lints all SQL files in the current project. To lint a specific SQL file or a directory, set `PATHS` to the path of the SQL file(s) or directory of files. To lint multiple files or directories, pass multiple `PATHS` flags.  
+
+To show detailed information on all the dbt supported commands and flags, run the `dbt sqlfluff -h` command. 
 
 #### Considerations
-Keep the following points in mind when using SQLFluff with the dbt Cloud:
 
-- When you run `dbt sqlfluff`, it picks up changes to your local .sqlfluff config.
-- To use SQLFluff in continuous integration/continuous development, you need to have a `dbt_cloud.yml` file in your project and run commands from a valid dbt project.
-- SQLFluff commands in the dbt Cloud CLI do not return exit codes yet.
+When running `dbt sqlfluff` from the dbt Cloud CLI, the following are important behaviors to consider:
+
+- dbt will read the `.sqlfluff` file if it exists for any custom configurations you might have.
+- For continuous integration/continuous development (CI/CD) workflows, your project must have a `dbt_cloud.yml` file and you have successfully run commands from within this dbt project.
+- An SQLFluff command will return an exit code of 0 if it ran without any file violations. This dbt behavior differs from SQLFluff behavior, where a linting violation returns a non-zero exit code. dbt Labs plans on addressing this in a later release.
```

Comment:
nit
```suggestion
- A SQLFluff command will return an exit code of 0 if it ran without any file violations. This dbt behavior differs from SQLFluff behavior, where a linting violation returns a non-zero exit code. dbt Labs plans on addressing this in a later release.
```

Reply:
i've been taught to use "Es cue el" vs "sequel" for SQL so "an" is correct

Reply:
I suppose this is a matter of taste. "An" reads a little strange to me and it looks like sqlfluff themselves are using "a sqlfluff" when writing similar sentences - see first bullet under highlights: https://github.com/sqlfluff/sqlfluff/blob/main/CHANGELOG.md#highlights-45

I'll defer to your best judgement though, feel free to proceed as you see fit 😃

Reply:
yeah, that's true, it's a pref thing. 

you know, i'll go ahead and add to our team meeting agenda this week. whatever we gavel on, i can update our style guide so we can all be consistent

Reply:
sounds good!